docs/SECURITY.md

```diff
@@ -1,12 +1,7 @@
 # Security Issues and Bugs
 
-Security issues can be reported by emailing jcappos@nyu.edu.
+Security issues can be reported to maintainers [privately via GitHub](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability):
 
-At a minimum, the report must contain the following:
-
-* Description of the vulnerability.
-* Steps to reproduce the issue.
-
-Optionally, reports that are emailed can be encrypted with PGP. You should use PGP key fingerprint **E9C0 59EC 0D32 64FA B35F 94AD 465B F9F6 F8EB 475A**.
+- [**Report new vulnerability**](https://github.com/theupdateframework/python-tuf/security/advisories/new)
```

Comment:
haha, so this Brand New Feature is actually just renaming the "New draft security advisory" button to a "report a vulnerability" button that's now visible to all. Well played Github